### PR TITLE
Allow to pass as UISchema options placeholder attribute for TextCell and TextAreaCell cells

### PR DESCRIPTION
**Pull request by andresgutgon**
# What?
Solve this issue https://github.com/eclipsesource/jsonforms/issues/1840

### CODE REVIEW

File: packages/examples/src/day2.ts [hunk 1]
```diff
@@ -32,7 +32,10 @@ export const uischema = {
   elements: [
     {
       type: 'Control',
-      scope: '#/properties/name'
+      scope: '#/properties/name',
+      options: {
+        placeholder: 'Placeholder for name'
+      }
```
Comment (sdirix) [inline packages/examples/src/day2.ts:38]: We should not modify the `day2` example. It is referred to [by a blog series](https://eclipsesource.com/blogs/2016/12/27/json-forms-day-2-introducing-the-ui-schema/).

File: packages/vanilla/src/cells/TextAreaCell.tsx [hunk 2]
```diff
@@ -44,6 +44,7 @@ export const TextAreaCell = (props: CellProps & VanillaRendererProps) => {
       id={id}
       disabled={!enabled}
       autoFocus={uischema.options && uischema.options.focus}
+      placeholder={uischema.options && uischema.options.placeholder}
```
Comment (sdirix) [inline packages/vanilla/src/cells/TextAreaCell.tsx:47]: As you're changing the code here anyway: We should use the same `appliedUiSchemaOptions` approach like in `TextCell.tsx`.

**Review by sdirix — CHANGES_REQUESTED** [review]
Thanks for the PR! Looks good and thanks for adding the test cases!

I only have minor comments.

File: packages/vanilla/src/cells/TextAreaCell.tsx (hunk 2 above)
Comment (andresgutgon) [reply packages/vanilla/src/cells/TextAreaCell.tsx:47]: Changing it now...

**Comment by andresgutgon** [comment]
Comments fixed : )

**Review by sdirix — APPROVED** [review]
LGTM! Thanks for the contribution!

**Comment by andresgutgon** [comment]
yay! thanks to you : )

**Comment by andresgutgon** [comment]
When something is merged is pushed to npm or do you have release days?

**Pull request merged**